javascript/javascript1/week3/javascript.js

```diff
@@ -0,0 +1,130 @@
+// Remove name from array list
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+for (var i = names.length - 1; i >= 0; i--) {
```

Comment:
i think when we use for,we have to use let instead of var

Reply:
Huhmm, You are right.. I am just going to update.